src/large_pages/node_large_page.cc

```diff
@@ -333,9 +333,6 @@ MoveTextRegionToLargePages(const text_region& r) {
     PrintSystemError(errno);
     return -1;
   }
-  OnScopeLeave munmap_on_return([nmem, size]() {
```

Comment:
This was the only conflict. It was expecting to remove `auto munmap_on_return = OnScopeLeave(...);` and not `OnScopeLeave munmap_on_return(...);`